.env.example

```diff
@@ -8,10 +8,16 @@ NEXT_PUBLIC_IPFS_BASE_URL=https://ipfs.io/ipfs/
 NEXT_PUBLIC_POH_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/kleros/proof-of-humanity-mainnet
 NEXT_PUBLIC_SISMO_GRAPH_API=https://api.sismo.io/
 NEXT_PUBLIC_SIGNATURE_API_URL=https://api.defender.openzeppelin.com/autotasks/4b1688f9-01a4-435d-89ae-d05e0aa0a53b/runs/webhook/b9818d77-3c43-4c3d-bfb8-4c77036de92f/ACXhXsQoCKP8zZVE26gFbh
+NEXT_PUBLIC_RPC_URL=https://rpc-mumbai.maticvigil.com/v1/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
maybe good to precise that it's only useful for backend: NEXT_PUBLIC_BACKEND_RPC_URL